website/data.yml

```diff
@@ -321,6 +321,18 @@ categories:
       repo_url: https://github.com/openai/codex
       tags:
       - coding
+    - name: crewAI
+      description: Framework for orchestrating role-playing, autonomous AI agents.
+        By fostering collaborative intelligence, CrewAI empowers agents to work together
+        seamlessly, tackling complex tasks.
+      homepage_url: https://www.crewai.com/
+      logo: crewai.svg
+      repo_url: https://github.com/crewAIInc/crewAI
+    - name: fast-agent
+      description: Define, Prompt and Test MCP enabled Agents and Workflows
+      homepage_url: https://github.com/evalstate/fast-agent
+      logo: fast-agent.png
```

Comment:
```suggestion
      logo: default.png
```

Reply:
I mean the author doesn't design a logo for this project, so by default it's using the author's logo. In this situation, we should use the default.png as the default placeholder.

Reply:
Anyway, we can merge this and change later.

Reply:
Can we use the default.png if no logo provides? @cr7258  Some projects don't have logos I believe.

Reply:
@kerthcet Ok, I'll take that into account.